### PR TITLE
Update keymap_spanish_dvorak.h

### PR DESCRIPTION
**Pull request by felixsanz**
#10211

## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: quantum/keymap_extras/keymap_spanish_dvorak.h [hunk 1]
```diff
@@ -119,7 +119,7 @@
 #define DV_CIRC S(DV_GRV)  // ^ (dead)
 #define DV_ASTR S(DV_PLUS) // *
 // Row 3
-#define DV_DIAE S(DV_GRV)  // ¨ (dead)
+#define DV_DIAE S(ES_ACUT)  // ¨ (dead)
```
Comment (fauxpark) [inline quantum/keymap_extras/keymap_spanish_dvorak.h:122]:
```suggestion
#define DV_DIAE S(DV_ACUT) // ¨ (dead)
```
And the `'` on line 29 should actually be `´`.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: quantum/keymap_extras/keymap_spanish_dvorak.h (hunk 1 above)
Comment (felixsanz) [reply quantum/keymap_extras/keymap_spanish_dvorak.h]: my bad, sorry!

**Review by drashna — APPROVED** [review]

**Review by fauxpark — CHANGES_REQUESTED** [review]
Could you also change the `'` on line 29 to `´`?

**Comment by fauxpark** [comment]
@felixsanz any updates here?

**Comment by felixsanz** [comment]
> @felixsanz any updates here?

sorry i don't know what to do here, i'm not a git user

**Comment by fauxpark** [comment]
@felixsanz see my previous review - the `'` character on line 29 needs to be changed to a `´` to match what the key actually outputs. You should be able to edit this from GitHub, I just can't add a code suggestion on it as it's too far out of the range of the existing changes here.

**Comment by felixsanz** [comment]
But both are already commited:

https://github.com/qmk/qmk_firmware/pull/10213/commits/52a91902ddad0eddb0a5b6f050751682c9f66c03
https://github.com/qmk/qmk_firmware/pull/10213/commits/d68dc7335b14a39f507e5f688f8d06efcb00aa10

**Comment by felixsanz** [comment]
Done

**Review by fauxpark — APPROVED** [review]

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**